SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -1159,7 +1159,7 @@ void CFVMFlowSolverBase<V, FlowRegime>::BC_Sym_Plane(CGeometry* geometry, CSolve
 
   /*--- Loop over all the vertices on this boundary marker. ---*/
 
-  SU2_OMP_FOR_DYN(OMP_MIN_SIZE)
+  //SU2_OMP_FOR_DYN(OMP_MIN_SIZE)
```

Comment:
What was the intent behind this change?

Reply:
I wanted to see what the effect was on the online regression tests. Sorry for the temporary confusion, I already changed it back :-)

Reply:
Word of cuation: If you remove a worksharing directive (instead of replacing, say, the dynamic schedule by a static one), the work is no longer shared, in the sense that each thread runs the full loop. This is usually not intended.